src/pages/DetailMemoryPage.js

```diff
@@ -1,54 +1,44 @@
 import styled from 'styled-components/macro'
-import BackButton from '../components/BackButton'
+import ToMemoriesButton from '../components/ToMemoriesButton'
 import PropTypes from 'prop-types'
 
 DetailMemoryPage.propTypes = {
-  image: PropTypes.node,
+  image: PropTypes.string,
+  title: PropTypes.string,
   alt: PropTypes.string,
 }
 
-export default function DetailMemoryPage({ image, onNavigate }) {
+export default function DetailMemoryPage({ image, title, onNavigate }) {
   return (
     <Wrapper>
       <ImageDetail>
-        <Image src={image} alt="Memory 1" width="100%" />
+        <Image src={image} alt="Memory" width="320" max-height="180" />
+        <span>{title}</span>
       </ImageDetail>
-      <nav>
-        <BackButtonStyle onClick={onNavigate}>Back to memories</BackButtonStyle>
-      </nav>
+      <ToMemoriesButton onClick={onNavigate} />
     </Wrapper>
   )
 }
 
 const Wrapper = styled.section`
   display: flex;
   flex-direction: column;
-  justify-items: center;
+  place-items: center;
   justify-content: space-between;
   height: 86vh;
-  width: 90%;
-
-  span {
-    letter-spacing: 4px;
-    font-size: larger;
-    text-shadow: 2px 2px 2px rgba(150, 150, 150, 1);
-  }
-`
-const Image = styled.img`
-  border-radius: 20px;
-  border-radius: 20px;
-  border: 4px;
-  border-color: white;
 `
-
 const ImageDetail = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   background-color: hsl(60, 2%, 80%, 0.8);
-  position: absolute;
-  left: 1.15em;
   padding: 10px;
-  width: 85%;
+  max-width: 88%;
```

Comment:
All those different % values looks like you should reorganize your height and width all over.